### DISCUSSION TITLE
Getting some weird text after exiting from editor page.

### DISCUSSION THREAD

**Discussion by asimex** [GrapesJS/grapesjs#5998] [Q&A]
I m getting some weird text after exiting from editor page to my other pages .

<img width="496" alt="Screenshot 2024-07-11 at 4 26 05 AM" src="https://github.com/GrapesJS/grapesjs/assets/38231806/8cefccae-12ea-40a8-9bf2-435c9d46dc04">

got this on inspecting the page 

<img width="593" alt="Screenshot 2024-07-11 at 4 25 37 AM" src="https://github.com/GrapesJS/grapesjs/assets/38231806/0246a2c5-ef39-4c18-a286-d1d302170c1d">

**Comment by artf** [comment]
Ensure to `editor.destroy()` when you unmount for another page

**Reply by asimex** [reply]
didnt worked ...in the end i have downgraded grapes to 0.21.8
